### PR TITLE
Don't remove source-linked assets (1.4)

### PR DESCRIPTION
**Pull request by boricj**
This prevents removing a feed when assets are using it.

### CODE REVIEW

**Comment by EldoreiJK** [comment]
This will leave from-links in DB, please fix this.

File: src/db/asset_general.cc [hunk 1]
```diff
@@ -615,7 +615,7 @@ db_reply_t
         return reply_delete2;
     }
 
-    auto reply_delete3 = delete_asset_links_all (conn, element_id);
+    auto reply_delete3 = delete_asset_links_to (conn, element_id);
```
Comment (EldoreiJK) [inline src/db/asset_general.cc]: This will leave from-links in DB, please fix this.

**Comment by boricj** [comment]
@EldoreiJK Please update your review following #497 discussion.

File: src/db/asset_general.cc (hunk 1 above)
Comment (EldoreiJK) [reply src/db/asset_general.cc]: Based on the discussion in master branch PR #497 I agree this should work in the current state of code, and should be enforced with (lib)fty-common-db split.

**Pull request merged**